h/static/scripts/annotator/main.js

```diff
@@ -13,54 +21,31 @@ Annotator.Plugin.CrossFrame.Bridge = require('../bridge')
 Annotator.Plugin.CrossFrame.AnnotationSync = require('../annotation-sync')
 Annotator.Plugin.CrossFrame.Discovery = require('../discovery')
 
-// Document plugin
-require('../vendor/annotator.document');
-
 // Bucket bar
 require('./plugin/bucket-bar');
 
 // Toolbar
 require('./plugin/toolbar');
 
-// Drawing highlights
-require('./plugin/texthighlights');
-
 // Creating selections
 require('./plugin/textselection');
 
-// URL fragments
-require('./plugin/fragmentselector');
-
-// Anchoring dependencies
-require('diff-match-patch')
-require('dom-text-mapper')
-require('dom-text-matcher')
-require('page-text-mapper-core')
-require('text-match-engines')
-
-// Anchoring plugins
-require('./plugin/enhancedanchoring');
-require('./plugin/domtextmapper');
-require('./plugin/fuzzytextanchors');
-require('./plugin/pdf');
-require('./plugin/textquote');
-require('./plugin/textposition');
-require('./plugin/textrange');
 
 var docs = 'https://h.readthedocs.org/en/latest/hacking/customized-embedding.html';
 var options = {
   app: jQuery('link[type="application/annotator+html"]').attr('href'),
-  BucketBar: {container: '.annotator-frame'},
+  BucketBar: {container: '.annotator-frame', scrollables: ['body']},
   Toolbar: {container: '.annotator-frame'}
 };
 
-// Simple IE autodetect function
-// See for example https://stackoverflow.com/questions/19999388/jquery-check-if-user-is-using-ie/21712356#21712356
-var ua = window.navigator.userAgent;
-if ((ua.indexOf("MSIE ") > 0) ||     // for IE <=10
-    (ua.indexOf('Trident/') > 0) ||  // for IE 11
-    (ua.indexOf('Edge/') > 0)) {     // for IE 12
-  options["DomTextMapper"] = {"skip": true}
+// Document metadata plugins
+if (window.PDFViewerApplication) {
```

Comment:
As much as possible I was trying to pull the document type switches out of the `Guest` and do it in one place, here. Probably, we can finish that by making `documentInfo` or something a first class citizen just as I did with the `anchoring` property.